src/main/java/com/yilmaznaslan/AppointmentFinder.java

```diff
@@ -17,19 +15,23 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.time.Duration;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
+import com.yilmaznaslan.formhandlers.Section1MainPageHandler;
+import com.yilmaznaslan.formhandlers.Section2ServiceSelectionHandler;
+import com.yilmaznaslan.formhandlers.Section3DateSelectionHandler;
+import com.yilmaznaslan.forms.PersonalInfoFormTO;
+import com.yilmaznaslan.forms.VisaFormTO;
+import com.yilmaznaslan.notification.NotificationAdapter;
+import com.yilmaznaslan.utils.DriverUtils;
+import com.yilmaznaslan.utils.IoUtils;
 
 public class AppointmentFinder {
 
+    public static final String FOREIGNERS_OFFICE_WEBSITE_HOMEPAGE = "https://otv.verwalt-berlin.de/ams/TerminBuchen?lang=en";
```

Comment:
Thats good but public modifier is not necessary, private should be enough

Reply:
That constant is indeed quite needed, as the Website URL is used all over the project.
I'm simply applying "Protected Variations".

This design pattern creates a single point of reference (and, thus, change) - next time it changes (because it will, and sooner than later), we just have to change it once 8-)

In fact, adding this paves the way to solving many already existing small bugs.

Reply:
> Thats good but public modifier is not necessary, private should be enough

Thanks. I will check the permissions.